src/js/components/OversiktspanelMedListe.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import HoyreChevron from 'nav-frontend-chevron';
+import PanelBase from 'nav-frontend-paneler';
+
+class OversiktspanelMedListe extends React.Component {
+  render() {
+    const { className, border, overskrift, ikon, headerLenkeTekst, headerLenkeHref, liste } = this.props;
+
+    return (
+      <PanelBase border={border} className={`oversiktspanel ${className}`}>
+        <div className="oversiktspanel__header">
+          <div className="oversiktspanel__overskrift">
+            { ikon
+              ? (
+                <div className="lenkepanel__ikon">
+                  {ikon}
+                </div>
+              )
+              : null}
+            <div className="typo-systemtittel">
+              {overskrift}
+            </div>
+          </div>
+          <div className="oversiktspanel__header-lenketekst typo-normaltekst">
+            <a href={headerLenkeHref} className="oversiktspanel__header-lenke" id="dekorator-bottomborder-overstyring">
+              {headerLenkeTekst}
```

Comment:
Fint om du alle steder med tekstlige lenker bruker <Lenke />-komponenten (https://design.nav.no/components/lenke) istedet for < a >, så får du med navs lenkestyling på kjøpet